code.py

```diff
@@ -1,9 +1,17 @@
 from purple.core import Core
 
-from purple.keyboard.mist.keyboard import keys
-from purple.keyboard.mist.layout.soar_left import Layout
+#from purple.keyboard.mist.keyboard import keys
+#from purple.keyboard.calc.keyboard import keys
+import purple.keyboard.calc.keyboard as kbd
+keys = kbd.keys
+use_led = kbd.use_led
+from purple.keyboard.calc.layout.simple_alphabet import Layout
 
 from purple.extras.performance_counter import PerformanceCounter
-from purple.extras.hardware.led import LED
+if use_led:
+    from purple.extras.hardware.led import LED
+    led = LED()
+else:
+    led = False
 
-Core(keys, Layout, LED(), PerformanceCounter()).run()
+Core(keys, Layout, led, PerformanceCounter()).run()
```

Comment:
Instead of setting **led** to False and passing it in here, only pass in the extras that you actually want to use:
```python
Core(keys, Layout, PerformanceCounter()).run()
```

Reply:
That makes sense!